src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +31,18 @@
      *                 В любом другом случае бросить DateTimeException
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        switch (datePart) {
+            case FULL:
+                return String.valueOf(LocalDate.now());
```

Comment:
let's create a variable and save the result of `LocalDate.now()` then we can use this variable at this method